pandas/_libs/tslibs/tzconversion.pyx

```diff
@@ -381,17 +378,15 @@ cpdef int64_t tz_convert_single(int64_t val, object tz1, object tz2):
         return _tz_convert_dst(arr, tz2, to_utc=False)[0]
 
 
-@cython.boundscheck(False)
-@cython.wraparound(False)
```

Comment:
Can we no longer guarantee the wraparound and bound check here?

Reply:
its just not needed, we're not doing any looping in this function